assets/info-page/index.html

```diff
@@ -0,0 +1,256 @@
+              <div class="select-icon">
+                <img src="../select-icon.png" alt="select-icon" class="d-none" id="select-icon__mm"/>
+              </div>
+            </div>
+            <div class="card__body">
+              <div class="card__sub-heading">What this choice means:</div>
+              <ul class="list">
+                <li class="list-item">
+                  <img src="../list-icon-info.png" alt="list-icon" class="list-icon">Linked channel and direct messages from Microsoft Teams will be synced in Mattermost
+                </li>
+                <li class="list-item">
+                  <img src="../list-icon-info.png" alt="list-icon" class="list-icon">You will get notifications in Mattermost for synced messages and channels.
+                </li>
+                <li class="list-item">
+                  <!-- TODO: Add correct links here -->
+                  <img src="../list-icon-warn.png" alt="list-icon" class="list-icon">You will need to disable notifications in Microsoft Teams to avoid duplicates. <a href="learn-more-url" class="list-link">Learn more</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="card color-none" onclick="selectPrimaryPlatform('msteams')" id="msteams">
+            <div class="card__heading">
+              <div class="card__logo-holder">
+                <img src="../msteams-icon.png" alt="msteams-icon" class="card__logo"/>
+              </div>
+              <div class="card__logo-details">Microsoft Teams will be my primary platform</div>
+              <div class="select-icon">
+                <img src="../select-icon.png" alt="select-icon" class="d-none" id="select-icon__msteams"/>
+              </div>
+            </div>
+            <div class="card__body">
+              <div class="card__sub-heading">What this choice means:</div>
+              <ul class="list">
+                <li class="list-item">
+                  <img src="../list-icon-info.png" alt="list-icon" class="list-icon">Linked channel and direct messages from Microsoft Teams will still be synced in Mattermost
+                </li>
+                <li class="list-item">
+                  <!-- TODO: Add correct links here -->
+                  <img src="../list-icon-warn.png" alt="list-icon" class="list-icon">Notifications in Mattermost will be muted for linked channels and DMs to prevent duplicates. <a href="learn-more-url" class="list-link">Learn more</a>
+                </li>
+                <li class="list-item">
+                  <img src="../list-icon-warn.png" alt="list-icon" class="list-icon">Unread statuses in linked channels and DMs will not be synced between Mattermost and Microsoft Teams.
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <div class="selection-button">
+          <!-- TODO: Update logic to select primary platform here -->
```

Comment:
This should call an endpoint in the plugin server side. What is the plan for that?

Reply:
added an endpoint to be called while selecting the primary platform